src/picture.rs

```diff
@@ -160,7 +160,7 @@ impl Rav1dPicAllocator {
                 w,
                 h,
                 layout: seq_hdr.layout,
-                bpc: 8 + 2 * seq_hdr.hbd,
+                bpc: 8 + 2 * seq_hdr.hbd as c_int,
```

Comment:
```suggestion
                bpc: 8 + 2 * seq_hdr.hbd,
```

We can change `Rav1dPictureParameters::bpc` to `u8` instead.